examples/tp+pp/pippy_tp.py

```diff
@@ -180,8 +158,7 @@ def main(args=None):
         "-s",
         "--schedule",
         type=str,
-        default=list(schedules.keys())[0],
-        choices=schedules.keys(),
+        default="FillDrain",
```

Comment:
N00b question what does this mean?

Reply:
Fill-drain is a pipeline scheduling policy. Other options include 1F1B, interleaved 1F1B. See for example [this paper](https://arxiv.org/pdf/2104.04473.pdf), section 2.2.